niworkflows/viz/utils.py

```diff
@@ -341,11 +341,16 @@ def plot_registration(
     if cuts is None:
         raise NotImplementedError  # TODO
 
+    anat_nii = _ensure_nii(anat_nii)
+
     out_files = []
     if estimate_brightness:
         plot_params = robust_set_limits(anat_nii.get_fdata().reshape(-1), plot_params)
 
     # FreeSurfer ribbon.mgz
+    if contour:
+        contour = _ensure_nii(contour)
```

Comment:
This is just `nb.Nifti1Image.from_image()`. Let's also add a comment so we know why we did this.

```suggestion
    # nilearn 0.10.0 uses Nifti-specific methods
    anat_nii = nb.Nifti1Image.from_image(anat_nii)

    out_files = []
    if estimate_brightness:
        plot_params = robust_set_limits(anat_nii.get_fdata().reshape(-1), plot_params)

    # FreeSurfer ribbon.mgz
    if contour:
        contour = nb.Nifti1Image.from_image(anat_nii)
```